sormas-app/app/src/main/java/de/symeda/sormas/app/backend/location/Location.java

```diff
@@ -333,6 +374,40 @@ public String getCompleteString() {
 			sb.append(getDetails());
 		}
 
+		if (getContactPersonFirstName()!=null || getContactPersonLastName()!=null){
```

Comment:
Code formatting

Also, first and last name should be combined (not separated by a comma), so you can basically tackle both in one if statement